t/sendreport/two-tier/email-and-open311.t

```diff
@@ -0,0 +1,379 @@
+use FixMyStreet::Cobrand;
+use FixMyStreet::Script::Reports;
+use FixMyStreet::TestMech;
+use Test::Deep;
+use Test::MockModule;
+use Test::MockObject;
+
+my $mech = FixMyStreet::TestMech->new;
+
+my $user = $mech->create_user_ok('user@example.com');
+
+my $body_oxf = $mech->create_body_ok( 2237, 'Oxfordshire County Council' );
```

Comment:
Given the mapit changes now on master, these will now need cobrand attributes adding.